azure-pipelines.yml

```diff
@@ -24,19 +18,21 @@ steps:
           versionSpec: 5.8.0
 
     - task: NuGetCommand@2
+      displayName: Restore NuGet packages for WinUI Gallery
       inputs:
         command: 'restore'
         restoreSolution: '**/*.sln'
         feedsToUse: 'config'
         nugetConfigPath: '$(System.DefaultWorkingDirectory)\nuget.config'
 
     - task: PowerShell@2
-      displayName: GenerateTestPFX.ps1
+      displayName: Generate test signing certificate
```

Comment:
nit: the display names should all have the first letter capitalized as it's a title.

Reply:
I've updated the titles now, but didn't capitalize words like "for". Hope thats fine.